Sources/Interpreter/Interpreter.swift

```diff
@@ -216,12 +162,16 @@ public struct Interpreter {
       _ = x
 
     case let x as AllocStack:
-      currentRegister = StackAllocation(typeLayout[x.allocatedType])
+      currentRegister = .address(allocate(typeLayout[x.allocatedType]))
 
     case let x as Branch:
-      _ = x
+      jumpTo(block: x.target)
+      return
     case let x as Call:
-      _ = x
+      stackFrames.append(
+        StackFrame(parameters: blockParams(from: x), returnAddress: try nextCodePointer()))
+      jumpToEntry(of: (x.callee.constant as! FunctionReference).function)
```

Comment:
```suggestion
      jumpTo((x.callee.constant as! FunctionReference).function)
```
It's the same as jumping to the function right?

Reply:
Yes!